ui-2/public/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+   
+    <title>Login</title>
+  </head>
+  <body>
+    <script src="../src/index.js" type="text/javascript"></script>
```

Comment:
all these "../src/<foo>" reference will need some work. for exemple change it to just "<foo>"

Reply:
sir I have to import the script form index.js